db/migrate/20220914231505_create_user.rb

```diff
@@ -1,10 +1,10 @@
 class CreateUser < ActiveRecord::Migration[7.0]
   def change
     create_table :users do |t|
-      t.string :Name
-      t.text :Photo
-      t.text :Bio
-      t.string :PostsCounter
+      t.string :name
+      t.text :photo
+      t.text :bio
+      t.string :postsCounter
```

Comment:
OPTIONAL

- [ ] Ruby naming convention requires that variable names should be in a snail_case. Instead of `postsCounter`, we should write `posts_counter`